src/psyclone/tests/test_files/gocean1p0/large_stencil.f90

```diff
@@ -0,0 +1,56 @@
+!-------------------------------------------------------------------------------
```

Comment:
Please replace this header with the standard one.

Reply:
Done. Note that I left you as author of this file, since the original code was yours, I only applied a very minor change (different kernel name) , and therefore (c) is also 2015-2021.

Reply:
Thanks.